cpp/src/io/utilities/datasource.cpp

```diff
@@ -102,6 +102,12 @@ class kvikio_source : public datasource {
     return supports_device_read();
   }
 
+  /**
+   * @param[in] dst Address of the existing device memory
```

Comment:
The comment should be in the doc of this declaration https://github.com/rapidsai/cudf/blob/branch-25.06/cpp/include/cudf/io/datasource.hpp#L303

Reply:
Addressed